docs/usage/schema.md

```diff
@@ -69,6 +69,23 @@ It has the following arguments:
   JSON Schema
 * `regex`: for string values, this adds a Regular Expression validation generated from the passed string and an
   annotation of `pattern` to the JSON Schema
+
+    !!! note
+        *pydantic* validates strings using `re.match`,
+        which treats regular expressions as implicitly anchored at the beginning.
+        On the contrary,
+        JSON Schema validators treat the `pattern` keyword as implicitly unanchored,
+        more like what `re.search` does.
+
+        For interoperability, depending on your desired behavior,
+        either explicitly anchor your regular expressions with `^`
+        (e.g. `^foo` to match any string starting with `foo`),
+        or explicitly allow an arbitrary prefix with `.*?`
+        (e.g. `.*?foo` to match any string containig the substring `foo`).
```

Comment:
Typo here: containig -> containing

Reply:
Sorry! Do I open a new PR or what?

Reply:
I guess so, only needs to be a tiny fix.